src/Exception/NotAllowedMethodException.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+namespace Opstalent\CrudBundle\Exception;
+
+/**
+ * Class NotAllowedMethodException
+ * @author Szymon Kunowski <szymon.kunowski@gmail.com>
+ * @package Opstalent\CrudBundle
+ */
+class NotAllowedMethodException extends \Exception
```

Comment:
is this part of this task?

Reply:
and why not `MethodNotAllowedException`?

Reply:
also it shouldn't extend `\Exception` it should extend `LogicException` or `RuntimeException`

Reply:
also: all our exceptions should implement our empty interface `Opstalent\CrudBundle\Exception\Exception` so there will be possibility of catching all exceptions from this bundle with one simple `try/catch`

Reply:
fixed